Controllers/controllers.js

```diff
@@ -33,3 +35,13 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+exports.getCommentsByArticle = (req, res, next) => {
+  const {article_id} = req.params
+  const { sort_by } = req.query;
+  return Promise.all([checkIfArticleExists(article_id),  selectCommentsByArticle(article_id,sort_by)])
```

Comment:
try swapping `checkIfArticleExists(article_id)`  to your model from ticket 5  `selectArticleById(article_id)`  in your `Promise.all` and see if this still works 

I believe it should still pass all tests and handle the errors the same - and then you can cut down on lines of code by removing the `checkIfArticleExists` function   ( sorry )